specification/std_extensions/collections.json

```diff
@@ -189,6 +189,22 @@
             }
           ],
           "output": [
+            {
```

Comment:
Seems wrong that we should have this file duplicated in source control, but that's another issue.